README.md

```diff
@@ -1,118 +1,80 @@
-# Jenkins Folder Properties Plugin
+A Jenkins plugin which allows users with config permission to define
+properties for a folder which can then be used by any jobs contained
+within it or any of its sub-folders. 
 
-A Jenkins plugin which allows users with config permission to define properties for a folder which can then be used by
-any jobs contained within it or any of its sub-folders.
```

Comment:
It is usually best for the maintainer if you do not "flow" or change the line breaks on existing text.  The change of line breaks on existing text makes it much more difficult to identify what has changed and what has remained the same.

Reply:
Thanks a lot for taking your time to review this @MarkEWaite !!. But, I didn't get what you meant by the _flow_ thing. Would you mind explaining a bit more?.

Reply:
> Thanks a lot for taking your time to review this @MarkEWaite !!. But, I didn't get what you meant by the _flow_ thing. Would you mind explaining a bit more?.

Sorry for the confusing phrasing.  When a sentence is word wrapped differently in a pull request than in the original, it appears as a complete line of changes in the GitHub diff output.  That makes the diff larger than necessary and more difficult for the reviewer to review.  I'm from a era when the word ["reflow"](https://www.techwalla.com/articles/what-is-text-reflow) was used to describe that style of text modification.

Reply:
haha, thanks for clarifying Mark!